netns_others.go

```diff
@@ -15,46 +15,46 @@ var (
 // is not implemented on other platforms.
 //
 // Deprecated: Use golang.org/x/sys/unix.Setns instead.
-func Setns(ns NsHandle, nstype int) (err error) {
+func Setns(_ NsHandle, _ int) error {
 	return ErrNotImplemented
 }
 
-func Set(ns NsHandle) (err error) {
+func Set(_ NsHandle) error {
 	return ErrNotImplemented
 }
 
-func New() (ns NsHandle, err error) {
+func New() (NsHandle, error) {
 	return -1, ErrNotImplemented
 }
 
-func NewNamed(name string) (NsHandle, error) {
```

Comment:
In general I prefer the `_` convention, however sometimes for primitive types, keeping the arg name is helpful... for example here I'd rather keep this one because even though unused, it clarifies to the caller what the arg is intended for. That way if we do ever implement it, callers are already ready for it. 

Same with all the rest of the following args in this file... 

Can you please add a per-line exception?